conductr_cli/conduct_logs.py

```diff
@@ -13,27 +15,93 @@ def logs(args):
 
     log = logging.getLogger(__name__)
 
+    if args.follow:
+        old_data = []
+
+        while not follow_is_done():
```

Comment:
This is a hook for testing - one of the tests has it return false after four executions.